### ISSUE TITLE
ClusterRunner should use hardcoded version string

### ISSUE THREAD

**Issue by shriganeshs** [box/ClusterRunner#348]
The version should be hardcoded string. If ClusterRunner could not find version in  cx_Freeze package, it assumes you're developing it locally and tries to calculate the version from git commits.
```bash
# clusterrunner -V
fatal: Not a git repository (or any of the parent directories): .git
ClusterRunner 0.5.???
```

Below is the code
```python
def get_version():
    """
    Get the version of the application. This method should return the correct version in both the frozen and unfrozen
    (running from cloned source) cases.
    :return: The version of the application
    :rtype: str
    """
    if getattr(sys, 'frozen', False):
        return _get_frozen_package_version()  # frozen/packaged

    return _calculate_source_version()  # unfrozen/running from source```

**Comment by josephharrington** [comment]
There are two tasks:
1. Execute logic that hardcodes the version before building the rpm.
2. Change the logic in `get_version` so that it doesn't rely on the `sys.frozen` attribute.

See setup.py for the logic that we'd need to execute before building the rpm:
```
version = autoversioning.get_version()
autoversioning.write_package_version_file(version)
```
This will create a file that essentially hardcodes the version number.

Just for background, the `getattr(sys, 'frozen', False)` pattern is not specific to cx_Freeze -- that is also used in other bundling/freezing tools.

But it sounds like bundling into an rpm does not set the `frozen` attribute, so we should decide on another way to detect it. Maybe it can switch on if it's running from a git repo or not. (If running in git repo then calculate the version, otherwise use hardcoded version.)

**Comment by josephharrington** [comment]
In case it was not clear from my above comment, my vote is to keep auto-versioning (not switch to manually-updated, hardcoded version numbers).

**Comment by shriganeshs** [comment]
Fixed in #352
